src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -867,6 +869,50 @@ private JsValue Raven_Min(JsValue self, JsValue[] args)
                 return args[0];
             }
 
+            private JsValue ArchivedAt(JsValue self, JsValue[] args)
```

Comment:
Please fix ArchivedAt -> ArchivdAt in all places

Reply:
i've pushed the amend